tests/EvmAcceptanceTests/test/GasEstimations.js

```diff
@@ -0,0 +1,34 @@
+const {expect} = require("chai");
+const {web3} = require("hardhat");
+const web3_helper = require("../helper/Web3Helper");
+const zilliqa_helper = require("../helper/ZilliqaHelper");
+
+describe("Gas estimation with web3.js", function () {
+  let contract;
+  const tranferFundMinGas = 21000;
+  const create2MinGas = 32000;
+
+  describe("When a fund transfer is made", function () {
+    it("should return proper estimation [@transactional]", async function () {
+      const gasAmount = await web3.eth.estimateGas({
+        to: zilliqa_helper.getSecondaryAccountAddress(),
```

Comment:
It's better to get all of your requirements from one of `zilliqa_helper` or `web3_helper`. Web3 helper does have this function for `primary account`. Just add another for `secondary` and use it.